### PR TITLE
fix: resolve spot price with the newest timestamp

### PR DESCRIPTION
**Pull request by jwcesign**
Fixes #none

## Proposed Changes

* fix: resolve spot price with the newest timestamp

**Release Note**

```release-note

```

### CODE REVIEW

File: pkg/client/alibabacloud.go [hunk 1]
```diff
@@ -113,13 +113,21 @@ func (a *AlibabaCloudPriceClient) Run(ctx context.Context) {
 	}
 }
 
+func getTargetFormatDate() string {
+	UTCLocation, _ := time.LoadLocation("UTC")
+	timeTarget := time.Now().In(UTCLocation)
+	return fmt.Sprintf("%04d-%02d-%02dT%02d:00:00Z",
+		timeTarget.Year(), timeTarget.Month(), timeTarget.Day(), timeTarget.Hour()-1)
```
Comment (jwcesign) [inline pkg/client/alibabacloud.go:120]: I scare if the current time is, like 14:00:1, if we use 14:00:00, there will be no entries, so, let's go with one hour earlier

File: pkg/client/alibabacloud.go [hunk 2]
```diff
@@ -113,13 +113,21 @@ func (a *AlibabaCloudPriceClient) Run(ctx context.Context) {
 	}
 }
 
+func getTargetFormatDate() string {
+	UTCLocation, _ := time.LoadLocation("UTC")
+	timeTarget := time.Now().In(UTCLocation)
```
Comment (Vacant2333) [inline pkg/client/alibabacloud.go:118]: timeTarget := time.Now().In(time.UTC)

File: pkg/client/alibabacloud.go (hunk 1 above)
Comment (Vacant2333) [inline pkg/client/alibabacloud.go:120]: we can leave a comment here,

**Review by Vacant2333 — COMMENTED** [review]
good good

**Pull request merged**